andela/dark-knight-rc
# #157911834 Cancel Order Option

#### What does this PR do?
It adds an option to enable the user to cancel an already placed order.

#### Description of Task to be completed?
- add a cancel button in the user's profile page
- enable the user to cancel an already placed order
- add a button in the admin page to enable the admin to remove an already canceled order

#### How should this be manually tested?
- after checking out and creating an order
- navigate to the profile page
- click on the cancel order button and you should then see a modal popup with a description of the status of your order
- click on accept
- on the admin invoice page, you should then see a button to remove the canceled order

#### Any background context you want to provide?
N/a

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/157911834

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
